### PR TITLE
Polish release notes 8.x

### PR DESCRIPTION
**Pull request by Jay-o-Way**
#8352

### CODE REVIEW

**Comment by mairaw** [comment]
Can you resolve the conflicts here too?

File: release-notes/8.0/install-windows.md [hunk 1]
```diff
@@ -27,11 +27,9 @@ Each of the distributions can be downloaded from:
...
 You can now install .NET updates using the Windows Package Manager CLI (winget):
 
-* To install the .NET 8 runtime: ```winget install dotnet-runtime-8```
-
-* To install the .NET 8 SDK: ```winget install dotnet-sdk-8```
-
-* To update an existing installation: ```winget upgrade```
+- To install the .NET 8 runtime: ```winget install dotnet-runtime-8```
+- To install the .NET 8 SDK: ```winget install dotnet-sdk-8```
+- To update an existing installation: ```winget upgrade```
```
Comment (mairaw) [inline release-notes/8.0/install-windows.md:32]:
```suggestion
- To install the .NET 8 runtime: `winget install dotnet-runtime-8`
- To install the .NET 8 SDK: `winget install dotnet-sdk-8`
- To update an existing installation: `winget upgrade`
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good. Just a suggestion for an improvement.

**Comment by mairaw** [comment]
I had already reviewed this PR and now it contain a bunch of extra files. Let's try to keep the PRs contained to avoid extra work.

**Comment by Jay-o-Way** [comment]
Sorry. Saw something that concerned files from the other, already merged, folders. You guys are quicker then I'm used to, here. Should I (try to) move the last commit to a new branch? Or pay attention to this next time?

**Comment by mairaw** [comment]
Just keep attention next time. Thanks!

**Review by mairaw — APPROVED** [review]

**Pull request merged**